### DISCUSSION TITLE
How to use isctl completion bash?

### DISCUSSION THREAD

**Discussion by mingjiga** [cgascoig/isctl#81] [Q&A]
Issuing "isctl completion bash" on my RHEL 8.5 server will output a shell script on the screen but auto completion still not working.

**Comment by cgascoig** [comment]
The shell script that it outputs needs to either be loaded into the current shell like this:
```
source <(isctl completion bash)
```

Or placed into the configuration for future new shells like this on Linux:
```
isctl completion bash > /etc/bash_completion.d/isctl
```

The help for the completion command has a little more detail as well:
```
isctl completion bash -h
```
